README.md

```diff
@@ -1,72 +1,116 @@
 
 <!-- DOCUMENTATION -->
 
 # Description
 
-- We are using multiple projects in one Angular environment so that we can help users to improve 
-  code  reusability, scalability, maintainability, and customization. It can also help in increase productivity, reduce risks, and improve the quality of  application.
+- The boilerplate using multiple projects in one Angular environment so that the boilerplate help users 
+  to improve code  reusability, scalability, maintainability, and customization. It can also help in increase productivity, reduce risks, and improve the quality of  application.
 
-- Code Reusability: I have multiple applications or parts of applications that share common functionality, 
-  using multiple projects can help you avoid duplicating code. You can create a shared library project and use it across all the projects. This approach can help you maintain consistency and reduce code duplication.
+- The multiple applications or parts of applications that share common functionality,  using multiple
+  projects can help you avoid duplicating code. You can create a shared library project and use it across all the projects. This approach can help you maintain consistency and reduce code duplication.
 
-- Isolation: Each project can be developed and tested independently, which allows you to isolate changes and
-  minimize the risk of breaking other parts of the application. This approach can help you increase productivity and reduce the time spent on debugging and fixing issues.
+Following are the steps to get started with it:
 
-- Scalability: As your application grows, it can become more complex and harder to maintain. Using multiple
-  projects can help you manage complexity by breaking down the application into smaller, more manageable pieces. You can also use different teams to work on different projects, which can help you scale the development process.
+# Prerequisite
 
-- Customization: Each project can have its own configuration and dependencies, which allows to 
-  customize the build process and optimize each project for its specific use case. This approach can help to improve the performance and reduce the size of the application.
+#### Step 1: Install NodeJS
+
+Install the latest LTS version from here: https://nodejs.org/en/download/.
+
+#### Step 2: Install Angular CLI
+
+Angular provides a very useful command line utility that help in easily developing angular applications, models, services, etc. as a boilerplate, which saves a lot of time.
+
+```sh
+npm install -g @angular/cli
+```
+
+#### step 3 : Install Nebular
+
+Similarly Angular also Provides Nebular which supports init configuration with Angular Schematics. This means you can simply add it to your project, and Angular Schematics will do the rest
+
+```sh
+ng add @nebular/theme
+```
+
+After this, it will take a few minutes to set everything up, once that is done, you will see a folder structure generated like below:
+
+```
```

Comment:
Why it needs to added manually, the boilerplate would already have it.

The steps to get started should be as easy as -
Clone the Repo
run npm install
and start the app... 

that's it.